File: javaPOO/POOAula03/src/aula02/Caneta.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package aula02;

public class Caneta {
    public String modelo;
    public String cor;
    private float ponta;
    protected int carga;
    private boolean tampada;
    
   public void status(){
        System.out.println("Modelo: " + this.modelo);
        System.out.println("Carga: " + this.carga);
        System.out.println("Tamanho da ponta: " + this.ponta);
        System.out.print("Uma caneta " + this.cor);
        System.out.println(" esta tampada? " + this.tampada);
        
    }
    public void rabiscar(){
        if (this.tampada == true){
            System.out.println("ERRO! Não posso rabiscar");  
        } else{
            System.out.println("Estou Rabiscando");
        }
    }
    
    public void tampar(){
        this.tampada = true;
    }
    
    public void destampar(){
        this.tampada = false;
    }
}

File: phpPOO/POOAula02/Caneta.php
<?php

class Caneta {
    var $modelo;
    var $cor;
    var $ponta;
    var $carga;
    var $tampada;
    
    function rabiscar(){
        if ($this->tampada == true) {
            print "<p>ERRO! Nao posso rabiscar!";
        }else{
        print "<p>Estou rabiscando</p>";
        }
    }
    function tampar(){
        $this-> tampada = true;
    }
    function destampar(){
        $this-> tampada = false;
    }
}
